osu.Server.Spectator/Hubs/StatefulUserHub.cs

```diff
@@ -33,15 +34,17 @@ protected StatefulUserHub(IDistributedCache cache)
 
         public override async Task OnConnectedAsync()
         {
-            Console.WriteLine($"User {CurrentContextUserId} connected!");
+            Log("Connected");
 
             try
             {
                 // if a previous connection is still present for the current user, we need to clean it up.
                 await cleanUpState(false);
             }
-            catch
+            catch (Exception e)
             {
+                Log("State cleanup failed");
```

Comment:
Did this mean to log the exception? The `e` local is not used.

Reply:
It's already logged via the `throw` below; i've removed the unaccessed variable.